src/components/organize/TagsManager/index.tsx

```diff
@@ -4,18 +4,28 @@ import { Box, Button, Popover, Typography } from '@material-ui/core';
 import { FormattedMessage, useIntl } from 'react-intl';
 
 import ZetkinSection from 'components/ZetkinSection';
-import { ZetkinTag } from 'types/zetkin';
 
 import GroupToggle from './GroupToggle';
+import { OnCreateTagHandler } from './types';
 import TagSelect from './TagSelect';
 import TagsList from './TagsList';
+import { ZetkinTag, ZetkinTagGroup } from 'types/zetkin';
 
 const TagsManager: React.FunctionComponent<{
```

Comment:
This could have an "assignOnCreate" boolean prop - to execute `onAssignTag` after `onCreateTag` for instances where this makes sense in the UI context

Reply:
for now i think i'll leave this outside the scope of the tags manager but that's a good point, maybe will refactor in the future. thank you.